randovania/games/prime1/logic_database/Phazon Mines.txt

```diff
@@ -626,9 +626,13 @@ Extra - aabb: [-86.10738, 118.43238, 15.926657, 29.835865, 233.99252, 83.92861]
                       Morph Ball Bomb
                       Power Bomb and Knowledge (Beginner)
   > Pickup (Missile)
-      All of the following:
-          After Elite Research Rock Wall
-          Space Jump Boots or Standable Terrain (Beginner)
+      Any of the following:
+          All of the following:
+              After Elite Research Rock Wall
+              Any of the following:
+                  Space Jump Boots or Combat/Scan Dash (Intermediate) or Standable Terrain (Beginner)
+                  Scan Visor and Combat/Scan Dash (Beginner)
+          Boost Ball and Morph Ball and Space Jump Boots and Clip Through Objects (Expert) and Single Room Out of Bounds (Expert) and Standable Terrain (Advanced)
```

Comment:
Can you also make it so that it requires either the event or another wall clip to escape from the item node?